models/autoencoder.py

```diff
@@ -0,0 +1,46 @@
+import sys
+
+import torch
+import torch.utils.data
+from torch import nn, optim
+from torch.nn import functional as F
+
+from data_loader import get_data_loaders
+from architectures import CNN_Decoder, CNN_Encoder
+
+sys.path.append("../")
```

Comment:
```suggestion
```

You should try to organize your modules so this is not needed. Maybe you can move the `architectures.py` file into the `models/` dir and do:

```python
from models.architectures import CNN_Decoder
```

Also, I think "architectures" is not the best here - maybe "layers" or "components" would be better?

Reply:
Changed